app/src/full/kotlin/io/homeassistant/companion/android/settings/wear/SettingsWearViewModel.kt

```diff
@@ -49,14 +54,62 @@ class SettingsWearViewModel @Inject constructor(private val serverManager: Serve
     AndroidViewModel(application),
```

Comment:
It's a bit outisde of the scope of the PR but could you remove the `@SuppressLint("VisibleForTests") // https://issuetracker.google.com/issues/239451111` since it is quite outdated.